fulp_modules/features/antagonists/bloodsuckers/code/bloodsucker/bloodsucker_datum.dm

```diff
@@ -334,8 +335,8 @@
 		if("join_clan")
 			if(my_clan)
 				return
+			ui.close()
 			assign_clan_and_bane()
-			ui.send_full_update(force = TRUE)
```

Comment:
The UI needs to be closed so that the radial menu is visible right after the "join clan" button is clicked. Once it is closed, it can't be easily reopened or updated without first either creating a new UI instance or doing something else that I'm completely unaware of.

Someone with a better understanding of TGUI might know how to go about doing this properly.